packages/brokeneck-react/src/components/Provider.js

```diff
@@ -1,17 +1,41 @@
 import React from 'react'
-import { Chip, makeStyles } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core'
 
+import Auth0Icon from '../icons/providers/auth0'
+import AzureIcon from '../icons/providers/azure'
+import CognitoIcon from '../icons/providers/cognito'
 import useProvider from '../hooks/useProvider'
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
+  icon: {
+    marginLeft: theme.spacing(1)
+  },
   text: {
+    marginLeft: theme.spacing(1),
     textTransform: 'capitalize'
+  },
+  wrapper: {
+    alignItems: 'center',
+    display: 'flex'
   }
-})
+}))
+
+const iconsMap = new Map()
+iconsMap.set('auth0', Auth0Icon)
+iconsMap.set('azure', AzureIcon)
+iconsMap.set('cognito', CognitoIcon)
```

Comment:
when I wrote a map I didn't actually mean a Map, more of a plain object lookup :) but this is perfectly fine too